src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/GetAdminTaskControllerCFTTest.java

```diff
@@ -85,7 +85,7 @@ public void should_return_a_200_with_allocateHearingJudge_task_and_standard_gran
             .body("task.case_name", notNullValue())
             .body("task.auto_assigned", notNullValue())
             .body("task.warnings", notNullValue())
-            .body("task.permissions.values", hasItems("Read", "Refer", "Own", "Manage", "Cancel"))
+            .body("task.permissions.values", hasItems("Read", "Own", "Manage", "Cancel"))
```

Comment:
Could we not use the values from the `PermissionTypes` enum here  ?

```
.body("task.permissions.values", hasItems(PermissionTypes.READ.value(),
PermissionTypes.OWN.value(),
PermissionTypes.MANAGE.value(),
PermissionTypes.CANCEL.value()))
```

if you used static imports, would make it less verbose as well

Reply:
Done.